test/Libraries/RevitIntegrationTests/WorkflowTest.cs

```diff
@@ -517,10 +517,12 @@ public void Test_Python()
             var pScript3 = "5a3b301c-632b-4ec1-9fcb-c2623f04c53c";
             AssertPreviewCount(pScript3, 2);
             var flatvalue3 = GetFlattenedPreviewValues(pScript3);
-            foreach (var ele in flatvalue3)
-            {
-                Assert.IsNotNull(ele);
-            }
+            Assert.IsTrue(flatvalue3.Contains("BoundingBox"));
+            Assert.IsTrue(flatvalue3.Contains("ByCurveAndHeight"));
+            Assert.IsTrue(flatvalue3.Contains("ByCurveAndLevels"));
+            Assert.IsTrue(flatvalue3.Contains("Curves"));
+            Assert.IsTrue(flatvalue3.Contains("Dispose"));
+            Assert.IsTrue(flatvalue3.Contains("Document"));
```

Comment:
The number may vary sometimes, what about to check if it is greater than 100. No need to get an actual number.

Reply:
is this the reason of number '6'..